redash/utils/parameterized_query.py

```diff
@@ -1,9 +1,34 @@
 import pystache
+from flask_login import current_user
 from numbers import Number
-from redash.utils import mustache_render
+from redash import models
+from redash.utils import mustache_render, json_loads
+from redash.permissions import require_access, view_only
 from funcy import distinct
 from dateutil.parser import parse
 
+
+def _pluck_name_and_value(row):
+    row = {k.lower(): v for k, v in row.items()}
+    name_column = "name" if "name" in row.keys() else row.keys()[-1]
+    value_column = "value" if "value" in row.keys() else row.keys()[-1]
+
+    return {"name": row[name_column], "value": row[value_column]}
+
+
+def _load_data(query_id, org):
+    query = models.Query.get_by_id_and_org(query_id, org)
+    require_access(query.data_source.groups, current_user, view_only)
```

Comment:
If you're using `current_user`, then I guess you can use `current_org` as well.

It does mean that we won't be able to run this outside the context of a Flask request, but at the moment I think there are no such cases?

Reply:
I didn't know there was a `current_org`. Yeah, I can't think of a case where we would use this outside a request right now. That's addressable if the need ever comes up.